src/app/services/image-store/image-store.service.ts

```diff
@@ -139,6 +148,35 @@ export class ImageStore {
     return thumbnails.find(thumb => thumb.imageIndex === index);
   }
 
+  private async getExtension(index: string) {
```

Comment:
The function name collides with `getExtension` from `mime-type.ts` so it's a bit confusing at the first glance because they do very different things.

Do you think `toExtension` would be a better name for the function in `mime-type.ts`? (Since another function is already called `fromExtension`)